components/Themes/src/MDCColorScheme.h

```diff
@@ -14,42 +14,92 @@
  limitations under the License.
  */
 
-#import <UIKit/UIKit.h>
-
 /**
  A color scheme comprised of set of primary and secondary colors. Material design guidelines
  recommend using primary and secondary colors with light and dark color variants.
  */
-@interface MDCColorScheme : NSObject
+@protocol MDCColorScheme
```

Comment:
See @willlarche comment on #1318
According to the new Swift API Design Guidelines:
https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage
Protocols that describe what something is should read as nouns (e.g. Collection).